tools/shared/src/main/scala/org/scalajs/core/tools/classpath/PartialClasspath.scala

```diff
@@ -151,6 +164,9 @@ object PartialClasspath {
   type DependencyFilter =
     Traversable[FlatJSDependency] => Traversable[FlatJSDependency]
 
+  type ManifestFilter =
+    Traversable[JSDependencyManifest] => Traversable[JSDependencyManifest]
```

Comment:
Well, you could at least make define it here as a type alias to `ManifestFilters.ManifestFilter`. There's no need for `jsdeps` to be aware of `classpath`.